lib/cat.js

```diff
@@ -9,16 +9,21 @@ const XLSX = require('xlsx')
 
 const {stringToStream} = require('./utils/stream')
 
-const getRows = async (fileOrStream) => {
+const getRows = async (fileOrStream, {sheet}={}) => {
+  let rows
   if (fileOrStream.constructor.name === 'Socket') {
-    return await toArray(fileOrStream.pipe(parse()))
+    rows = await toArray(fileOrStream.pipe(parse()))
   } else {
-    return await toArray(await fileOrStream.rows())
+    rows = await toArray(await fileOrStream.rows({sheet}))
+  }
+  if (!rows || rows.length === 0) {
+    throw new Error('Input source is empty or doesn\'t exist.\n> If you\'re using excel file, remember that sheet index starts from 1. You can also use a sheet name.')
```

Comment:
What else may user be using besides excel, when using this command?

Reply:
@zelima any tabular file